restcomm/restcomm.commons/src/main/java/org/restcomm/connect/commons/util/DigestAuthentication.java

```diff
@@ -46,31 +46,53 @@ private static String A1(final String algorithm, final String user, final String
     }
 
     private static String A2(final String method, final String uri, String body, final String qop) {
+        return A2("MD5", method, uri, body, qop);
```

Comment:
@abdulazizali77 why do we hardcode to MD5 instead of using client-algorithm from the settings ?